Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,155 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
```

Comment:
It'll be great to add some context on the important changes - from a customer who is viewing this workbook perspective, it'll be great to provide a quick overview of workbooks getting covered here and then the important note. 
Please rephrase the text to be 'CEF standard custom label functionality has limitations and to solve those, here are the field mappings done for CyberArk data connector. Refer to the table below for further context.'